winlogbeat/scripts/mage/config.go

```diff
@@ -0,0 +1,60 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This feels quite a bit hidden in `scripts` package but not something to address for this PR, more a general comment.